vcluster/configure/vcluster-yaml/logging.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Logging
+sidebar_label: logging
+sidebar_position: 13
+description: Configuration for setting structured logging options for vCluster.
+---
+
+import InterpolatedCodeBlock from '@site/src/components/InterpolatedCodeBlock'
+import Logging from '../../_partials/config/logging.mdx'
+
+vCluster outputs structured logs that help you monitor and troubleshoot your virtual clusters. You can configure the log format to suit your monitoring and analysis tools and workflows.
+
+## Configure log encoding
+
+You can set the log encoding format in your `vcluster.yaml` configuration:
+
+<InterpolatedCodeBlock 
```

Comment:
This doesn't actually do the indentation properly (not sure why), but considering there are only two options, I don't think we need to use this type of block.